Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/model/SingleAchievement.kt

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+package org.dolphinemu.dolphinemu.features.settings.model
+
+class SingleAchievement(
+    val title : String,
+    val description : String,
+    val badgeName : String,
+    val measuredProgress : String,
+    val measuredPercent : Float,
+    val id : Int,
+    val points : Int,
+    val unlockTime : String,
+    val state : Int,
+    val category : Int,
+    val bucket : Int,
+    val unlocked : Int,
+    val rarity : Float,
+    val rarityHardcore : Float,
+    val type : Int,
+    val badgeUrl : String,
+    val badgeLockedUrl : String)
```

Comment:
You're putting a space on both sides of the colon in a lot of your code. There should only be a space after, not before.